src/ray/raylet/worker.cc

```diff
@@ -15,7 +15,14 @@ Worker::Worker(pid_t pid, std::shared_ptr<LocalClientConnection> connection)
     : pid_(pid),
       connection_(connection),
       assigned_task_id_(TaskID::nil()),
-      actor_id_(ActorID::nil()) {}
+      actor_id_(ActorID::nil()),
+      blocked_(false) {}
+
+void Worker::MarkBlocked() { blocked_ = true; }
```

Comment:
minor: `ToggleBlocked(true/false)` to save on methods. But we can clean it up if we decide to go with worker states later.